src/components/cards/Card/Card.tsx

```diff
@@ -26,17 +23,13 @@ export const Card = ({ title, date, userName, status }: PropsCardType) => {
           <p className='m-0'>{title}</p>
           <p className='m-0'>
             <span className='text-text-primaryAlpha300'>By </span>
-            {userName}
+            {author.name}
           </p>
         </div>
         <div className='flex flex-col text-center items-center gap-4'>
           <Link href='/testPage'>
             <Button>{t('start')}</Button>
           </Link>
-          <div>
-            <span className='text-text-primaryAlpha300'>Created: </span>
```

Comment:
why did you remove 'Created" ?

Reply:
because this field is not in the data received from the server

{"id": 1,
"title": "You Dont know NodeJS",
"authorId": 7,
"author": {
"id": 7,
"name": "asdasd",
"surname": null,
"email": "sdad2@mail.ru",
"nickname": "asasdsaa",
"password": "$2b$10$UZ6KUgDmp3N2MsibblSikOaVFP0rQ2NcMNYlglnI4sU0e.RU/XSnq",
"status": null
},
"question": []
}